RELEASE_NOTES

```diff
@@ -1,3 +1,14 @@
+=======================================================================
+amazon-cloudwatch-observability v2.0.0 (2024-08-15)
+========================================================================
+Enhancements:
+* Allow configurable resource requests and limits through command arguments for AppSignals (#65)
```

Comment:
I think we want to mention that we have a breaking change section to say we have enforced default requests and limits for auto instrumentation init containers. Similar to the opentelemetry-collector releases: https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.107.0


```
Breaking Changes:
* Enforce default requests and limits for auto instrumentation init containers
Enhancements:
* Allow configurable requests and limits for auto instrumentation init containers
```

Reply:
+1 to that. additionally lets capture that as part of the helm-chart release as well

Reply:
yes, I was going to adding those breaking changes to Github release page while keeping the release notes format as-is, but I agree that they are better to be in sync.